### PR TITLE
Add index file and upgrade to sdk 1.0.0

### PR DESCRIPTION
**Pull request by charlieduong94**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -450,17 +450,10 @@
...
-"@jupiterone/integration-sdk@^0.17.0":
-  version "0.17.0"
-  resolved "https://registry.yarnpkg.com/@jupiterone/integration-sdk/-/integration-sdk-0.17.0.tgz#fc88a00367df2ef84c7c95bf881f8fa81cc2fc4b"
-  integrity sha512-R326bc5AFgHrDCUM3+qLKPqEtywmSAHlEEJwxqEjUM1X6IqlEGkJenXURSLXZpJUoA2ErblkNGJSHfK1r6Ei4g==
-=======
-"@jupiterone/integration-sdk@^0.15.0":
-  version "0.15.0"
-  resolved "https://registry.yarnpkg.com/@jupiterone/integration-sdk/-/integration-sdk-0.15.0.tgz#865141491ff45a68c55d3ae998e3d6265b26d410"
-  integrity sha512-ys071w/Eevv+nRGa81sdnb93ln8Xv+iIHSNOxZqCM9NGsUT7oEuMyNOz2CnzHxRjj7FUj6kZucug1bVPJW5KHg==
->>>>>>> 1b0cf4b45eeeedeed983d1321ed08b6b0caccf63
```
Comment (charlieduong94) [inline yarn.lock:463]: I'm really surprised that builds passed with conflicts in the yarn.lock...

Comment (aiwilliams) [reply yarn.lock:463]: Yah, WUT 😕

File: package.json [hunk 2]
```diff
@@ -20,10 +22,10 @@
     "test:ci": "yarn lint && yarn type-check && yarn test",
     "prebuild": "yarn test:ci",
     "build": "tsc -p tsconfig.dist.json --declaration",
-    "prepublishOnly": "yarn build"
+    "prepack": "yarn build"
```
Comment (aiwilliams) [inline package.json:25]: This is a very nice change, I think, IIUC https://yarnpkg.com/cli/pack and https://yarnpkg.com/advanced/lifecycle-scripts. 💯

File: package.json [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
   "name": "@jupiterone/graph-cisco-meraki",
-  "version": "1.1.2",
+  "version": "1.2.0",
   "description": "A graph conversion tool for https://meraki.cisco.com",
   "license": "MPL-2.0",
   "files": [
     "dist"
   ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```
Comment (aiwilliams) [inline package.json:10]: I also love these.

**Pull request merged**